frontend/javascripts/admin/api/jobs.ts

```diff
@@ -74,6 +74,20 @@ export async function cancelJob(jobId: string): Promise<APIJob> {
   });
 }
 
+export type JobCostInfo = {
+  costInCredits: string;
```

Comment:
how is the cost encoded in the string? can you add a comment?

Reply:
e.g., "The cost is encoded as a string decimal for precision reasons. The front-end should not do any arithmetics with this."